lib/srv/desktop/rdp/rdpclient/src/lib.rs

```diff
@@ -1900,3 +1900,5 @@ pub(crate) type Messages = Vec<Message>;
 pub(crate) trait Encode: std::fmt::Debug {
     fn encode(&self) -> RdpResult<Message>;
 }
+
+const MAX_RDP_MESSAGE_SIZE: usize = 2_usize.pow(21); // 2MB
```

Comment:
The name here suggests that the RDP protocol has a maximum message size, but this is more of an arbitrary value that we selected, so let's use a better name here.